### ISSUE TITLE
Quicksave "lag spike"

### ISSUE THREAD

**Issue by OCircles** [shinyquagsire23/OpenJKDF2#46]
There is always a lag spike on quicksaves, and the longer you go between quicksaves, the bigger the lag spike gets (I've had it be like 3s or so?)

Interestingly, it's more of a "freeze" than a legit lag spike, because the game *does* continue to run, it's just that the renderer just gets stuck showing you the same frame for a while.

You don't have to play for long without quicksaving to start experiencing quite significant ones

**Comment by shinyquagsire23** [comment]
Yeah, confirmed on MacOS as well, though it oddly only seems to happen once. If I load a savefile for Into The Dark Palace and then quicksave I get a 2s spike.
